pyccel/parser/semantic.py

```diff
@@ -3292,7 +3291,6 @@ def _visit_FunctionDef(self, expr, **settings):
                         methods, superclass=cls.superclass))
 
             funcs += [func]
-
```

Comment:
Please try to avoid committing files where you have only changed the whitespace. It adds things to review while not actually doing anything helpful (everyone has a different opinion on what whitespace choices are the most readable). To undo these changes you can run:
```
git checkout master -- pyccel/parser/semantic.py
```

Reply:
Same comment as last time

Reply:
I overlooked this, I ll change it asap.